disassemble/disassemble.cpp

```diff
@@ -48,30 +51,47 @@ int main(int argc, char** argv) {
     cout << error;
     return -1;
   }
-  auto fit = all.begin();
-  Function* f = *fit;
   // create an Instruction decoder which will convert the binary opcodes to strings
-  InstructionDecoder decoder(f->isrc()->getPtrToInstruction(f->addr()),
-                             InstructionDecoder::maxInstructionLength, f->region()->getArch());
-  for(; fit != all.end(); ++fit) {
+  InstructionDecoder decoder((const void *)nullptr, 1, sts->getArch());
```

Comment:
The cast is needed here to avoid ambiguity between `InstructionDecoder(const void *, size_t, Architecture)` and `InstructionDecoder(const unsigned char *, size_t, Architecture)` It doesn't really matter which constructor will be used, but the compiler doesn't know this and throws a error.

Reply:
Yet another violation of the Principle of Least Astonishment... I will fix the redundant constructor overload.